src/Unirest/Request.php

```diff
@@ -474,13 +490,15 @@ public static function buildHTTPCurlQuery($data, $parent = false)
         foreach ($data as $key => $value) {
             if ($parent) {
                 $new_key = sprintf('%s[%s]', $parent, $key);
-            } else {
+            }
+            else {
```

Comment:
undo all these else-block moves and refactoring changes that are done in this PR